src/LightPanels/LightPanels.js

```diff
@@ -192,7 +192,6 @@ module.exports = kind(
 			this.updateTransforms();
 			this.orientationChanged();
 			this.directionChanged();
-			this.indexChanged();
```

Comment:
It probably should have changed to a call to `setupTransitions()` rather than removing it since that was effectively what was happening before.